dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -118,7 +118,8 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
 
   test 'teacher cannot view nonexistent section details' do
     sign_in @teacher
-    get :show, params: {id: 1_000_000}
+    nonexistant_id = Section.last.id + 1
```

Comment:
If there are other unit tests running in parallel (and some of them might not be doing a good job of isolating state), maybe we should add a larger number like 1000 or even 1000000 just to make it extra-nonexistent?